.yamllint

```diff
@@ -0,0 +1,29 @@
+---
+
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+  - '.yamllint'
+
+rules:
+  braces: enable
+  brackets: disable
+  colons: enable
+  commas: enable
+  comments:
+    level: warning
+  comments-indentation:
+    level: warning
+  document-end: disable
+  empty-lines: enable
+  empty-values: disable
+  hyphens: enable
+  key-duplicates: enable
+  key-ordering: disable
+  line-length: disable
+  new-line-at-end-of-file: enable
+  new-lines: enable
+  octal-values: disable
+  quoted-strings: disable
+  trailing-spaces: enable
+  truthy: disable
```

Comment:
```suggestion
  truthy:
    allowed-values: ['True', 'False']
```
We shouldn't disable this, If we want capitalised True/False as we currently use then set that

Reply:
let me check on my PR whether this works in form of `true` / `false`

Reply:
works in "native yaml" bool value
https://github.com/sqlfluff/sqlfluff/pull/2154/checks

Reply:
Yup just changed in https://github.com/sqlfluff/sqlfluff/pull/2162/commits/2cafb66cbb822479e1ae686995e8a498ba24c46e. As discussed on slack:
> Just change them all to lowercase in that PR after fighting with too many exceptions for other (non-SQLFluff) files to allow us to keep True with initial uppercase
> It’s a bit inconsistent since Python insists on True and now standardising on true for YAML but seems the simplest